include/Token/Token.h

```diff
@@ -39,6 +39,7 @@ class Token {
 public:
   void setIndex(llvm::GetElementPtrInst *GEPInst);
   void setIndex(llvm::GEPOperator *GEPOp);
+  void setIndex(std::string); //Added on 31.5.21
```

Comment:
What are the use-cases where someone would manually update the index?

Reply:
> What are the use-cases where someone would manually update the index?
We have discussed and resolved this query.